src/symbolic/symbolic_wasm_ffi.ml

```diff
@@ -48,6 +48,8 @@ module M :
 
   let symbol_f64 = symbol (Ty_fp 64)
 
+  let symbol_bool = symbol (Ty_bitv 32)
+
```

Comment:
@Laplace-Demon, here I believe you have to constrain the symbol further otherwise it can be any `int32`